packages/storybook/src/nimble/anchor/anchor.stories.ts

```diff
@@ -47,7 +47,7 @@ const metadata: Meta<AnchorArgs> = {
             options: Object.keys(AnchorAppearance),
             control: { type: 'radio' },
             description:
-                'Set to `prominent` to make the anchor appear in a different color than normal text.'
+                'Set to `prominent` to make the anchor appear in a different color than normal text. This has no effect under the Color theme.'
```

Comment:
nit: while you're here would you mind adding a style to set the color of the non-link text using a token? It looks bad in dark mode right now. 
<img width="212" alt="image" src="https://github.com/ni/nimble/assets/10500124/3227d5d7-d24f-4381-9f3e-0a180c6e31c8">